### PR TITLE
Partial rix for alt.hololive.tv

### PR DESCRIPTION
**Pull request by AndrewDaGuy**
Things-not-fixed-yet-list:
- The corners of the lower part is white
image

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -604,6 +604,16 @@ INVERT
 
 ================================
 
+alt.hololive.tv
+
+INVERT
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:609]: Very loosy selectors. Unless your absolute sere that it won't mess up other things, try to "hardened" it a bit.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -604,6 +604,16 @@ INVERT
...
+video[src*="bg.mp4"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:610]: Or there is being something appended to this otherwise you can also replace the `*` to `$`

Comment (AndrewDaGuy) [reply src/config/dynamic-theme-fixes.config:610]: There's something appended to it `?20210215165645` (Seems like a upload date or something `? 2021 02 15 165645`)

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:610]: Got it!

**Pull request merged**